src/Coinbase/Spot/API/Candle.jl

```diff
@@ -81,12 +86,12 @@ to_pretty_json(result.result)
 ]
 ```
 """
-function candle(client::CoinbaseClient, query::CandleQuery; product_id::String)
-    return APIsRequest{Vector{CandleData}}("GET", "products/$product_id/candles", query)(client)
+function candle(client::CoinbaseClient, query::CandleQuery;)
+    return APIsRequest{Vector{CandleData}}("GET", "products/$(query.product_id)/candles", query)(client)
 end
 
-function candle(client::CoinbaseClient = Coinbase.Spot.public_client; product_id::String, kw...)
-    return candle(client, CandleQuery(; kw...); product_id = product_id)
+function candle(client::CoinbaseClient = Coinbase.Spot.public_client; kw...)
+    return candle(client, CandleQuery(; kw...);)
```

Comment:
```suggestion
    return candle(client, CandleQuery(; kw...))
```
`;` It will be superfluous here